tests/test_e2e_31_of_lldp.py

```diff
@@ -0,0 +1,249 @@
+        api_url = f"{KYTOS_API}/of_lldp/v1/liveness/"
+        response = requests.get(api_url)
+        data = response.json()
+        for entry in data["interfaces"]:
+            assert entry["id"] in interface_ids, entry
+            assert entry["status"] == "up", entry
+
+    def test_002_disable_link_liveness(self) -> None:
+        """Test disable link liveness."""
+        polling_interval = 1
+        self.set_polling_time(polling_interval)
+        interface_ids = [
+            "00:00:00:00:00:00:00:03:2",
+            "00:00:00:00:00:00:00:02:3",
+            "00:00:00:00:00:00:00:01:3",
+            "00:00:00:00:00:00:00:02:2",
+            "00:00:00:00:00:00:00:01:4",
+            "00:00:00:00:00:00:00:03:3",
+        ]
+        self.enable_link_liveness(interface_ids)
+
+        # Wait just so liveness can go up
+        time.sleep(polling_interval * 5)
+
+        # Assert GET liveness/ is enabled
+        api_url = f"{KYTOS_API}/of_lldp/v1/liveness/"
+        response = requests.get(api_url)
+        data = response.json()
+        for entry in data["interfaces"]:
+            assert entry["id"] in interface_ids, entry
+            assert entry["status"] == "up", entry
+
+        self.disable_link_liveness(interface_ids)
+
+        # Assert GET liveness/ is disabled
+        api_url = f"{KYTOS_API}/of_lldp/v1/liveness/"
+        response = requests.get(api_url)
+        data = response.json()
+        assert data["interfaces"] == []
+
+        # Restart the controller maintaining config
+        self.restart(wait_for=10)
+
+        # Assert GET liveness/ is still disabled
+        api_url = f"{KYTOS_API}/of_lldp/v1/liveness/"
+        response = requests.get(api_url)
+        data = response.json()
+        assert data["interfaces"] == []
+
+    def test_005_liveness_goes_down(self) -> None:
+        """Test liveness goes down."""
+        polling_interval = 1
+        self.set_polling_time(polling_interval)
+        interface_ids = [
+            "00:00:00:00:00:00:00:01:3",
+            "00:00:00:00:00:00:00:02:2",
+        ]
+        link_id = "78282c4d5b579265f04ebadc4405ca1b49628eb1d684bb45e5d0607fa8b713d0"
+        self.enable_link_liveness(interface_ids)
+
+        # Wait just so liveness can go up
+        time.sleep(polling_interval * 5)
+
+        # Assert GET liveness/ is enabled
+        api_url = f"{KYTOS_API}/of_lldp/v1/liveness/"
+        response = requests.get(api_url)
+        data = response.json()
+        for entry in data["interfaces"]:
+            assert entry["id"] in interface_ids, entry
+            assert entry["status"] == "up", entry
+
+        """
+        Install a high prio LLDP flow to drop just so packet-ins
+        are no longer sent. This will force hello to be missed.
+        """
+        s2 = self.net.net.get("s2")
+        flow_entry = "cookie=0xdd00000000000000,priority=30000,dl_type=0x088cc,dl_vlan=3799,actions=drop"
+        s2.dpctl("add-flow", flow_entry)
```

Comment:
A very interesting and wise solution to use a flow_entry to simulate the loss! Just for future reference, we can also leverage the loss property from Mininet's TCIntf (http://mininet.org/api/classmininet_1_1link_1_1TCIntf.html#a3854309a926ae4bc40f4fc15895838fa) to accomplish such behavior. An example could be:
```
s2 = self.net.net.get("s2")
for i in s2.intfs:
    s2.intfs[i].link.intf1.config(loss=100.0)
    s2.intfs[i].link.intf2.config(loss=100.0)
```

Then, after the test, to recover the link forwarding:
```
s2 = self.net.net.get("s2")
for i in s2.intfs:
    s2.intfs[i].link.intf1.config(loss=0.000001)
    s2.intfs[i].link.intf2.config(loss=0.000001)
```

Reply:
Thanks for your review @italovalcy. Good to know that this functionality exists, seems very handy, thanks for sharing.